### PR TITLE
Room filter

### PR DESCRIPTION
**Pull request by yashka713**
Added and realised room-filter

Realised filter dor room by `price', `pccupancy`, `room type` and `date`

### CODE REVIEW

File: app/controllers/homes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'ApartmentsFilterForm.rb'
```
Comment (Vitaliy-Maznyi) [inline app/controllers/homes_controller.rb:1]: why do you need to require this file?

File: app/models/ApartmentsFilterForm.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  private
+
+  def start_from_date_not_in_past
+    return unless start_from < Date.today
+    errors.add :start_from, 'start date cannot be in past'
+  end
+
+  def price_from
+    price = :price
```
Comment (Vitaliy-Maznyi) [inline app/models/ApartmentsFilterForm.rb]: Why do you do this on line 24?

Comment (yashka713) [reply app/models/ApartmentsFilterForm.rb]: Because `:price.to_s` is not working. And method `split` absent for symbol.

Comment (Vitaliy-Maznyi) [reply app/models/ApartmentsFilterForm.rb]: why can't you use just `price`? Try to comment this line and test behaviour

**Pull request merged**